import-dashboards.py

```diff
@@ -35,6 +35,8 @@
 LOGO_FILE                  = '/usr/share/pmm-server/landing-page/img/pmm-logo.svg'
 SET_OF_TAGS                = {'Query Analytics': 0, 'OS': 0, 'MySQL': 0, 'MongoDB': 0, 'PostgreSQL': 0, 'Insight': 0, 'PMM': 0}
 YEAR                       = str(datetime.date.today())[:4]
+DBAAS                      = os.getenv('PERCONA_TEST_DBAAS') is not None
```

Comment:
`PERCONA_TEST_DBAAS` can be `0` or `False`

Reply:
I have extended the function for checking all possible values of the variable.